_data/components/form.yml

```diff
@@ -156,7 +156,7 @@ outputs: [{
   name: "beforeCloseDetail",
   description: "Event triggered before leaving form detail mode"
 },{
-  name: "beforeGoEditMode",
+  name: "beforeGoEditMode (deprecated)",
```

Comment:
Maybe, it would be great to point the new method that have to be used

Reply:
Added to description